flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ForStFlinkFileSystem.java

```diff
@@ -42,10 +44,16 @@ public class ForStFlinkFileSystem extends FileSystem {
     // TODO: make it configurable
     private static final int DEFAULT_INPUT_STREAM_CAPACITY = 32;
 
+    private static final String SST_SUFFIX = ".sst";
```

Comment:
Could this be a `Function<String, Boolean> filter`? I mean a file filter for local path, a member for each instance, and a default static filter for SST.

Reply:
👍Changed.